sentry.client.config.ts

```diff
@@ -0,0 +1,12 @@
+import { init, Replay } from '@sentry/nextjs';
+
+import { SENTRY_DSN } from '@/next.constants.mjs';
+
+init({
+  dsn: SENTRY_DSN,
+  tracesSampleRate: 1,
+  debug: false,
```

Comment:
this is the default https://docs.sentry.io/platforms/javascript/guides/nextjs/configuration/options/#debug
```suggestion
```